hebiros/include/hebiros/hebiros.cpp

```diff
@@ -208,8 +208,290 @@ bool Hebiros_Node::srv_add_group_from_urdf(
   return Hebiros_Node::srv_add_group_from_names(names_req, names_res);
 }
 
-//Subscriber callback which receives a joint state and sends that as a command to a group
-void Hebiros_Node::sub_command(const boost::shared_ptr<sensor_msgs::JointState const> data,
+//Add joint state values to a group command
+void Hebiros_Node::add_joint_command(GroupCommand* group_command,
+  sensor_msgs::JointState data, std::string group_name) {
```

Comment:
This changes the behavior a bit from before -- I just want to make sure this is deliberate.  Before, any module that wasn't explicitly named was sent a default "nan" command; now, there is no command sent to any module not named.  I think either one of these behaviors is reasonable as long as we are clear about the behavior.